sdk/cognitivelanguage/azure-ai-language-conversations/samples/async/sample_analyze_conversation_app_async.py

```diff
@@ -49,53 +42,60 @@ async def sample_analyze_conversation_app_async():
     client = ConversationAnalysisClient(clu_endpoint, AzureKeyCredential(clu_key))
     async with client:
         query = "Send an email to Carol about the tomorrow's demo"
-        result = await client.analyze_conversation(
-                task=CustomConversationalTask(
-                    analysis_input=ConversationAnalysisOptions(
-                        conversation_item=TextConversationItem(
-                            text=query
-                        )
-                    ),
-                    parameters=CustomConversationTaskParameters(
-                        project_name=project_name,
-                        deployment_name=deployment_name
-                    )
-                )
-            )
-
-        # view result
-        print("query: {}".format(result.results.query))
-        print("project kind: {}\n".format(result.results.prediction.project_kind))
-
-        print("top intent: {}".format(result.results.prediction.top_intent))
-        print("category: {}".format(result.results.prediction.intents[0].category))
-        print("confidence score: {}\n".format(result.results.prediction.intents[0].confidence))
-
-        print("entities:")
-        for entity in result.results.prediction.entities:
-            print("\ncategory: {}".format(entity.category))
-            print("text: {}".format(entity.text))
-            print("confidence score: {}".format(entity.confidence))
-            if entity.resolutions:
-                print("resolutions")
-                for resolution in entity.resolutions:
-                    print("kind: {}".format(resolution.resolution_kind))
-                    print("value: {}".format(resolution.additional_properties["value"]))
-            if entity.extra_information:
-                print("extra info")
-                for data in entity.extra_information:
-                    print("kind: {}".format(data.extra_information_kind))
-                    if data.extra_information_kind == "ListKey":
-                        print("key: {}".format(data.key))
-                    if data.extra_information_kind == "EntitySubtype":
-                        print("value: {}".format(data.value))
-
-    # [END analyze_conversation_app_async]
+        result = await client.conversation_analysis.analyze_conversation(
```

Comment:
I'm hoping that once we regenerate then this will just become `client.analyze_conversation(...)` but let me know if there is still an issue